### ISSUE TITLE
Project Index page

### ISSUE THREAD

**Issue by clairerye** [ebi-ait/dcp-ingest-central#117]
## User Story
As a 'researcher with a computer' I want a central point to access all data relevant for HCA research
As a stakeholder I want to know how much HCA data is available and stratify it for various purposes
As a 'researcher with a pipette' I want my data shared with the community

## Suggested requirements to support the data release goals:
Contributors and researchers must have a central catalog showing the full breadth of the data at a project level (individual data file level is not required). 
Researchers and collaborators must be able to obtain access to the data, but this can be via direct download or link out to appropriate archives/store for metadata 
Ideally, researchers and collaborators should be able to search the catalog to identify projects of interest (for example, filtering by organ, technology, date) 

## Nice to have
Researchers and collaborators should be able to see a “project view” summary page giving a high level overview of the project and links to data

**Comment by mshadbolt** [comment]
proposed potential URLs:
```
http://ebi.ac.uk/human-cell-atlas/projects 
http://ebi.ac.uk/human-cell-atlas/project-catalog
http://ebi.ac.uk/human-cell-atlas/project-index

http://ebi.ac.uk/hca/projects - Gabby
http://ebi.ac.uk/hca/project-catalog
http://ebi.ac.uk/hca/project-index
```

**Comment by mshadbolt** [comment]
contacted web prod to get `ebi.ac.uk/humancellatlas` domain - ticket: `www-prod #472626`

**Comment by mshadbolt** [comment]
@jacobwindsor I added 'supplementary_links' to the `data.json`, I am not really sure the best way to display them, I guess we would just need to use the link text as the display text because it can contain any kind of link

**Comment by mshadbolt** [comment]
I have added all HCA pubs and various others that seem appropriate from the tracker.

I will do another review to see if there are any more in ingest that aren't appearing.

**Comment by mshadbolt** [comment]
General status update:
- web prod are working on hosting the site at the url we chose, should be live in the next day or so
- solr implementation is still chugging along, @jacobwindsor doing a lot of reading and learning to understand how to set up. Will be set up on ingest infrastructure and will need edits to the API in order to make functional
- We decided not to implement pagination for now, this will need to be implemented when we are querying the ingest API rather than loading from a json
- @gabsie & @jacobwindsor to set up a meeting to go through small ui tweaks together 
- google analytics - we need to decide exactly WHAT we want to track, but we are bit blocked on full implementation until we have the final URL. @gabsie to let @jacobwindsor know the account details for the ingest GA account when he starts working on that. Can use some of the basic setup already implemented in ingest. (detailed discussion on #140)
- We decided to avoid complication of having extra pages we could have a short description at the top of the page that is expandable, gabs and I to work on text then provide to @jacobwindsor (#126)
- @gabsie to provide some input on the metadata properties, the embl ones seem a bit weird and not overly relevant but Gabs to review and ensure we are providing the important ones for google search etc (#139 )

If I forgot anything in the above please add !

**Comment by tburdett** [comment]
Great summary, thanks. I can add a view on the goal of what we want to track.

The goal (from the UX research plan) says our aim is to:

> Provide scientists that are trying to find all HCA data assets with a trustable and comprehensive list of available data from all projects and publications that contribute to building the Human Cell Atlas

For me, this means I would expect users to follow a "data discovery" journey. For example, a user might search for some data by a criteria they expect results for (e.g. tissue type), review the results, and then follow links out to archives in order to download datasets that interest them. We should be able to demonstrate this by a) checking what people search for and b) detecting when they click through on an archive link. We will be doing a good job if users follow links to archives for a high number of search results returned.

This is just my thoughts, though - totally open to input if there are other workflows we expect that I am missing.
